src/hooks.js

```diff
@@ -56,18 +56,22 @@ export function useArrows({ onUp, onLeft, onDown, onRight } = {}) {
 
 function stepsReducer(state, { type, value, steps }) {
   const { step } = state
+  const stepsCount = steps - 1
 
   let newStep = null
 
   if (type === 'set') {
     newStep = value
   }
-  if (type === 'next' && step < steps - 1) {
+  if (type === 'next' && step < stepsCount) {
     newStep = step + 1
   }
   if (type === 'prev' && step > 0) {
     newStep = step - 1
   }
+  if (type === 'remember') {
```

Comment:
Perhaps another name is `'adjustSize'`; in the context of this function itself, `'remember'` is somewhat disconnected from the context.

Reply:
Yeah totally, that's much better.